stream-chat-android-client/src/main/java/io/getstream/chat/android/client/StreamLifecycleObserver.kt

```diff
@@ -10,19 +10,27 @@ import kotlinx.coroutines.launch
 
 internal class StreamLifecycleObserver(private val handler: LifecycleHandler) : LifecycleObserver {
     private var recurringResumeEvent = false
+    @Volatile private var isObserving = false
+    private val coroutineScope = CoroutineScope(DispatcherProvider.Main)
```

Comment:
If this scope is never cancelled, we might as well use `GlobalScope` instead (and specify a dispatcher in the coroutine builder, e.g. `launch`), because it makes no difference, and that's simpler.

Reply:
:+1: